Modules/ThirdParty/MetaIO/src/MetaIO/src/localMetaConfiguration.h

```diff
@@ -57,5 +61,18 @@
 #    define METAIO_EXPORT
 #  endif
 
-// end VTK/ITK
+#else
+// Independent of ITK and VTK
+
+#  define METAIO_USE_NAMESPACE 0
+#  define METAIO_NAMESPACE metaio
+#  define METAIO_STREAM std
+
+#  include "itk_zlib.h"
```

Comment:
I think this include needs to be in the ITK section above.

Reply:
This change should be made in MetaIO itself.

Reply:
This is confusing, but correct.   The configuration for ITK is handled by the first conditional (#if) statement.  The #elif conditional handles the VTK configuration, and this third (#else) conditional if when MetaIO is built independent of ITK or VTK.  The confusion is the #include of itk_zlib.h - which is done for convenience since zlib is packaged with MetaIO using that name mangling when downloaded from the main repo (https://github.com/Kitware/MetaIO/tree/master/itkzlib).   It looks odd in here, but it is correct and necessary - you just need to see the main MetaIO repo for it to make sense.

Reply:
> This change should be made in MetaIO itself.

The code will not compile stand alone (outside of ITK and VTK) without this.